styles/index.css

```diff
@@ -68,7 +62,7 @@
     border-radius: 10%;
 }
 
-.categories img:hover {
+.categories img:hover, .card:hover {
     box-shadow: rgba(256, 256, 256, 0.35) 10px 10px 15px;
```

Comment:
use color variable

Reply:
Color variable does not work inside box property: https://stackoverflow.com/questions/59250927/box-shadow-with-rgba-variable